Libraries/Modal/Modal.js

```diff
@@ -218,7 +205,12 @@ class Modal extends React.Component<Props> {
 
     const innerChildren = __DEV__ ? (
       <AppContainer rootTag={this.context.rootTag}>
```

Comment:
I think this will be broken now because `this.context.rootTag` will be undefined because you removed 

```
 static contextTypes: any | {|rootTag: React$PropType$Primitive<number>|} = {
    rootTag: PropTypes.number,
  };
```
I think you'll need to leave that part until https://github.com/facebook/react-native/pull/28104 is landed.

Reply:
You actually don't need to wait, there is already a RootTagContext you can use: https://github.com/facebook/react-native/commit/f9ecce7440e712782f4fd760a46e54b3102914d8